app/src/main/java/com/kickstarter/ui/toolbars/DiscoveryToolbar.java

```diff
@@ -67,6 +69,11 @@ protected void onFinishInflate() {
       final Context context = getContext();
       context.startActivity(new Intent(context, ActivityFeedActivity.class));
     });
+
+    creatorDashboardButton.setOnClickListener(v -> {
```

Comment:
I wonder if this and `activityFeedButton` can use butter knife's `@OnClick`, like the menu button later in the code?